client/my-sites/plugins/main.jsx

```diff
@@ -384,8 +400,9 @@ export default React.createClass( {
 		];
 	},
 
-	formatPlugins( plugins ) {
+	formatPlugins( plugins, isWpCom ) {
 		const manageableSites = this.props.sites.getSelectedOrAllJetpackCanManage();
+		const inBulkEditMode = this.state.bulkManagement.indexOf( isWpCom ? 'wpcom' : 'jetpack' ) !== -1 ? true : false;
```

Comment:
Hmmm, I may be alone here, but I rather prefer 
```
const inBulkEditMode = this.state.bulkManagement.indexOf( isWpCom ? 'wpcom' : 'jetpack' ) !== -1;
// (let's remove the non needed ternary operator)
```
 over 
```
const inBulkEditMode = ! includes( bulkManagement, isWpCom : 'wpcom' : 'jetpack' );
```
Because yeah, the syntax is nicer using includes, but the first approach is readable for anyone speaking vanilla js, and the second one forces you to check (or know in advance) the lodash docs (and yeah, its part of ES7, so sooner or later everybody will need to be used to it .. but right now it still means to add a 3rd party lib, and to be honest, the legibility improvement over using .indexOf() > -1 is pretty marginal IMHO)